bootstrap-salt.sh

```diff
@@ -2352,7 +2374,7 @@ install_debian_6_deps() {
     fi
 
     # shellcheck disable=SC2086
-    wget $_WGET_ARGS -q http://debian.saltstack.com/debian-salt-team-joehealy.gpg.key -O - | apt-key add - || return 1
+    __fetch_verify http://debian.saltstack.com/debian-salt-team-joehealy.gpg.key 267d1f152d0cc94b23eb4c6993ba3d67 3100 | apt-key add - || return 1
```

Comment:
You're welcome! In cases like these I hate to give a review which seems pedant/bureaucratic, but in reality if you want to be able to say "my deployment is safe against MiTM" each and every piece must satisfy that requirement (that's where I come from).